contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,282 @@
+    sum(CASE WHEN event ='important button clicked' THEN 1 ELSE 0 END) as event_1,
+    sum(CASE WHEN event = 'thing saved to db' THEN 1 ELSE 0 end) as event_2,
+    sum(CASE WHEN event = 'team member invited' THEN 1 ELSE 0 end) as event_3
+  FROM events
+  WHERE toDate(timestamp) >= now() - interval 6 month
+  -- filter by the events here
+  AND event IN ('important button clicked', 'thing saved to db', 'team member invited')
+  GROUP BY 1,2
+), successful_companies AS (
+  SELECT
+    starting_companies.company_id
+  from starting_companies
+  join downfunnel on starting_companies.company_id = downfunnel.company_id
+  where downfunnel.date_str >= starting_companies.start_date
+  and downfunnel.date_str <= starting_companies.start_date + interval 30 day -- this interval is the amount of time someone has to complete all the activation events. you can try moving it up or down.
+  group by 1
+  -- below, define the number of times you want each event to happen. Sometimes people have to do an event 2 or 3 times before they "get it."
+  having sum(coalesce(downfunnel.event_1, 0)) >= 1 AS event_1
+  AND sum(coalesce(downfunnel.event_2, 0)) >= 2 AS event_2
+  AND sum(coalesce(downfunnel.event_3, 0)) >= 1 AS event_3
+),
+retained AS (
+  SELECT
+    DISTINCT e.person.properties.company_id as company_id
+  FROM events e
+  JOIN starting_companies ON e.person.properties.company_id = starting_companies.company_id
+  WHERE toDate(timestamp) >= starting_companies.start_date + interval 3 month
+  and toDate(timestamp) <= starting_companies.start_date + interval 4 month
+  -- below, any events that tell you that the user/company is still using the product 3 months later
+  AND event IN ('certain page viewed', 'thing saved to db', 'button clicked', 'logged in')
+)
+SELECT
+    starting_companies.start_month,
+    COUNT(DISTINCT starting_companies.company_id) as total_starting_companies,
+    COUNT(DISTINCT successful_companies.company_id) as total_companies_activated,
+    round((COUNT(DISTINCT successful_companies.company_id) * 100.0 / coalesce(COUNT(DISTINCT starting_companies.company_id), 0)), 2) as activation_percentage,
+    COUNT(DISTINCT CASE WHEN retained.company_id IS NOT NULL THEN successful_companies.company_id else null END) as total_activated_companies_survived,
+    round((COUNT(DISTINCT retained.company_id) * 100.0 / coalesce(COUNT(DISTINCT successful_companies.company_id), 0)), 2) as retained_percentage_of_activated
+FROM starting_companies
+LEFT JOIN successful_companies ON starting_companies.company_id = successful_companies.company_id
+LEFT JOIN retained ON successful_companies.company_id = retained.company_id
+GROUP BY starting_companies.start_month
+ORDER BY starting_companies.start_month
+```
+</details>
+
+## What do I do with the activation metric once I find it?
+
+You push for it, of course! If you know you the actions you want people to do, you can:
```

Comment:
I think "push for it" is the wrong term here. So is my suggestion ("improving them"). What we're really trying to say is that you focus your product efforts in increasing these metrics. I dont know what the elegant way to say this is
```suggestion
You work on improving them, of course! If you know you the actions you want people to do, you can:
```